src/main/java/fi/tkgwf/ruuvi/config/Config.java

```diff
@@ -419,7 +419,7 @@ public static long getMeasurementUpdateLimit() {
     }
 
     public static boolean isAllowedMAC(String mac) {
-        return filterMode.test(mac);
+        return mac!=null &&  filterMode.test(mac);
```

Comment:
The common convention here is to have spaces around all operators, including the `!=`